autogen/agentchat/conversable_agent.py

```diff
@@ -139,20 +140,14 @@ def __init__(
             else (lambda x: content_str(x.get("content")) == "TERMINATE")
         )
 
-        if llm_config is False:
+        if llm_config is False or llm_config is None and self.DEFAULT_CONFIG is None:
```

Comment:
When would DEFAULT_CONFIG not be None here?

Reply:
Is DEFAULT_CONFIG class or instance scoped? Either they'd have to subclass it or change it for all usages of the class. It doesn't feel very easy to understand